PublicationsRouter/api/v3/Config.md

```diff
@@ -24,6 +24,12 @@ If authentication was successful then you will receive a 200 (OK) and a JSON doc
         ...
     }
 
+Else, if authentication was unsucessful you will receive a 401 (Unauthorized) 
```

Comment:
-1- Better to say "Otherwise, ..."

-2- Will a 401 be returned with a JSON data structure? If not, then move this new text after line 33 (which presumably goes with the 200 result).